orchagent/request_parser.h

```diff
@@ -0,0 +1,114 @@
+#ifndef __REQUEST_PARSER_H
+#define __REQUEST_PARSER_H
+
+typedef enum _request_types_t
+{
+    REQ_T_BOOL,
+    REQ_T_STRING,
+    REQ_T_MAC_ADDRESS,
+    REQ_T_PACKET_ACTION,
+    REQ_T_NOT_USED,
+} request_types_t;
+
+typedef struct _request_description
+{
+    std::vector<request_types_t> key_item_types;
+    std::unordered_map<std::string, request_types_t> attr_item_types;
+    std::vector<std::string> mandatory_attr_items;
```

Comment:
>std::vector<std::string> mandatory_attr_items; [](start = 4, length = 46)

I am ok with current implementation. 

but, it would be more extensible to add different property for the attribute item, for example, the type, whether it is mandatory or not, whether it depends on other attribute item of not.